03/app.js

```diff
@@ -0,0 +1,13 @@
+const buttons = document.querySelectorAll("button");
+
+buttons.forEach((e) => {
+	const showText = function () {
+		this.textContent = "clicked";
+		console.log(this.textContent);
+
+		if (this.textContent === "clicked") {
```

Comment:
Czy `if` jest tutaj potrzebny? Skoro za pierwszym razem mamy odpiąć nasłuchiwanie to czy musi być `if`?